templates/controllers_deployment.yml

```diff
@@ -47,6 +50,27 @@ spec:
         volumeMounts:
           - name: uaa-client-secret
             mountPath: /config/uaa_client_secret
+      - name: backup-metadata-generator
+        image: #@ data.values.images.backup_metadata
+        imagePullPolicy: Always
+        command: [ "/cnb/process/wait" ]
```

Comment:
I'm guessing this is just to keep the container indefinitely alive, right?

Reply:
Yeah, it's defined as `sleep infinity` in the `Procfile`:
https://github.com/cloudfoundry/capi-k8s-release/blob/d1487cd15cb8da2e202d0a53504531a33216f051/src/backup-metadata/Procfile#L1

Previously this was a bash while/sleep loop:
https://github.com/cloudfoundry/capi-k8s-release/blob/35a5bc67cc04404176db490e1ab5c251c29aab96/src/backup-metadata/config/cfmetadata.yml#L28-L29